lib/releaselib.sh

```diff
@@ -57,11 +57,11 @@ release::update_job_cache () {
     # the cache should be complete after that
     [[ -n ${JOB[$run]} ]] && break
```

Comment:
The JOB dictionary is initially populated with the current cache in $job_file.  
$run starts at the latest build and indexes $JOB up to $cache_limit. 
When ${JOB[$run]} is set, it breaks.